notebooks/Lab3_LinearRegression/bike-sharing-demand-regression.ipynb

```diff
@@ -3159,7 +3159,7 @@
    "source": [
     "**Question** Calculate the RMSE. Use the cell below.\n",
     "\n",
-    "Note: numpy has a linear algebra package containing a function for calculating the "
+    "Note: numpy has a linear algebra package containing a function for calculating the !!!"
```

Comment:
I was not sure how to finish this statement. My guess is RMSE? But I could not find the package online.